notifications/alertchannelapimethods.go

```diff
@@ -0,0 +1,13 @@
+package notifications
+
+func (ac *AlertChannelAPI) GetDomainScope() []AlertScope {
+	scope := make([]AlertScope, 0, len(ac.Scope))
+
+	for _, enrichedScope := range ac.Scope {
+		scope = append(scope, AlertScope{
```

Comment:
avoid append and allocate the array before

Reply:
wdym?